aws/default_retryer_test.go

```diff
@@ -152,7 +154,7 @@ func TestGetRetryDelay(t *testing.T) {
 	}
 
 	for i, c := range cases {
-		a, ok := getRetryDelay(&c.r)
+		a, ok := getRetryAfterDelay(&c.r)
```

Comment:
The test function for this test case should be renamed from `TestCanUseRetryAfter` to `TestGetRetryAfterDelay`